scripts/netlify/build-website.sh

```diff
@@ -4,6 +4,6 @@ set -e
 
 $(npm bin)/lerna exec --scope @feature-hub/website -- rm -rf build
 $(npm bin)/lerna exec --scope @feature-hub/website -- docusaurus-build
-$(npm bin)/lerna exec --no-private --parallel -- typedoc --options ../../typedoc.js --out ../../packages/website/build/feature-hub/\$LERNA_PACKAGE_NAME --tsconfig tsconfig.json .
-node scripts/netlify/process-api-docs.js
+$(npm bin)/typedoc --options typedoc.js .
+node scripts/process-api-docs.js
```

Comment:
why duplicate the script invocation here / in the package.json ?

Reply:
using `yarn` now